### ISSUE TITLE
Caret position is buggy when using 'Hide currency symbol'

### ISSUE THREAD

**Issue by blowsie** [dm4t2/vue-currency-input#274]
### Vue Currency Input version

2.0.2

### Vue version

3.1.5

### What browser are you using?

Edge

### What operating system are you using?

Windows

### Reproduction repo/CodeSandbox

https://dm4t2.github.io/vue-currency-input/playground.html

### Describe your issue

When using the following settings the caret position varies:
```
{
	locale: undefined,
	currency: 'EUR',
	currencyDisplay: 'hidden',
	valueRange: undefined,
	precision: undefined,
	hideCurrencySymbolOnFocus: true,
	hideGroupingSeparatorOnFocus: false,
	hideNegligibleDecimalDigitsOnFocus: false,
	autoDecimalDigits: false,
	exportValueAsInteger: false,
	autoSign: false,
	useGrouping: false
}
```

Sometimes the caret is at end, sometime it is at one from end

![T1m3T30N3r](https://user-images.githubusercontent.com/308572/133090777-fb1701cb-5483-47f2-889b-6ce251f95129.gif)

**Comment by blowsie** [comment]
The desired result can be achieved by setting `hideCurrencySymbolOnFocus`, but I think it's a bug none the less

**Comment by dm4t2** [comment]
Thanks for reporting 👍

**Comment by dm4t2** [comment]
Fixed in [2.1.0](https://github.com/dm4t2/vue-currency-input/releases/tag/2.1.0) 🎉

**Comment by blowsie** [comment]
⚡  That was fast! Thanks.
